src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax;
+
+public class Square extends AbstractFigure {
+    private double side;
+
+    public Square(String color, double side) {
+        super(color);
+        this.side = side;
+    }
+
+    @Override
+    public double getArea() {
+        return side * side;
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("Figure: square, area: " + getArea() + " sq. units, side: " + side + " units, color: " + getColor());
```

Comment:
Instead of directly using `System.out.println` within the `draw` method, consider using a separate class or method to handle output, enhancing the separation of concerns.